src/pymor/models/iosys.py

```diff
@@ -929,6 +929,376 @@ def get_ast_spectrum(self, ast_pole_data=None, mu=None):
+        solver_options
+            The solver options to use to solve the Lyapunov equations.
+        name
+            Name of the system.
+
+        Attributes
+        ----------
+        order
+            The order of the system.
+        dim_input
+            The number of inputs.
+        dim_output
+            The number of outputs.
+        J
+            The |Operator| J.
+        R
+            The |Operator| R.
+        G
+            The |Operator| G.
+        P
+            The |Operator| P.
+        S
+            The |Operator| S.
+        N
+            The |Operator| N.
+        E
+            The |Operator| E.
+        transfer_function
+            The transfer function.
+        """
+
+    def __init__(self, J, R, G, P, S=None, N=None, E=None, solver_options=None,
+                 error_estimator=None, visualizer=None, name=None):
+        assert J.linear
+        assert J.source == J.range
+
+        assert R.linear
+        assert R.source == J.source
+        assert R.source == R.range
+
+        assert G.linear
+        assert G.range == J.source
+
+        assert P.linear
+        assert P.range == J.source
+        assert P.source == G.source
+
+        S = S or ZeroOperator(G.range, G.source)
+        assert S.linear
+        assert S.source == G.source
+        assert S.range == S.source
+
+        N = N or ZeroOperator(G.range, G.source)
+        assert N.linear
+        assert N.source == G.source
+        assert N.range == N.source
+
+        E = E or IdentityOperator(J.source)
+        assert E.linear
+        assert E.source == E.range
+        assert E.source == J.source
+
+        assert solver_options is None or solver_options.keys() <= {'lyap_lrcf', 'lyap_dense'}
+
+        super().__init__(dim_input=G.source.dim, error_estimator=error_estimator, visualizer=visualizer, name=name)
+        self.__auto_init(locals())
+        self.solution_space = J.source
+        self.dim_output = G.source.dim
+
+        K = lambda s: s * self.E - (self.J - self.R)
+        B = lambda s: self.G - self.P
+        C = lambda s: (self.G + self.P).H
+        D = lambda s: self.S - self.N
+        dK = lambda s: self.E
+        dB = lambda s: ZeroOperator(self.G.range, self.G.source)
+        dC = lambda s: ZeroOperator(self.G.source, self.G.range)
+        dD = lambda s: ZeroOperator(self.S.range, self.S.source)
+        parameters = Parameters.of(self.J, self.R, self.G, self.P, self.S, self.N, self.E)
+
+        self.transfer_function = FactorizedTransferFunction(
+            self.dim_input, self.dim_output,
+            K, B, C, D, dK, dB, dC, dD,
+            parameters=parameters, name=self.name + '_transfer_function')
+
+    @classmethod
+    def from_matrices(cls, J, R, G, P, S=None, N=None, E=None,
+                      state_id='STATE', solver_options=None, error_estimator=None,
+                      visualizer=None, name=None):
+        """Create |PHLTIModel| from matrices.
```

Comment:
To use `|PHLTIModel|`, a substitution needs to be defined in `docs/source/substitutions.py`.